GVFS/GVFS/CommandLine/CloneVerb.cs

```diff
@@ -125,6 +125,12 @@ public override void Execute()
                     cloneResult = this.TryCreateEnlistment(fullEnlistmentRootPathParameter, normalizedEnlistmentRootPath, out enlistment);
                     if (cloneResult.Success)
                     {
+                        // Create the enlistment root explicitly with CreateDirectoryAccessibleByAuthUsers before calling
+                        // AddLogFileEventListener to ensure that elevated and non-elevated users have access to the root.
+                        // But first create the ancestors of the enlistment root with the default ACLs (in case they're not on disk already)
+                        Directory.CreateDirectory(Path.GetDirectoryName(enlistment.EnlistmentRoot));

```

Comment:
Again, `CreateDirectoryAccessibleByAuthUsers` already does this, no?

Reply:
Good catch here as well, I will fix this up.